### DISCUSSION TITLE
helmRelease pruning

### DISCUSSION THREAD

**Discussion by iacou** [fluxcd/flux2#1041] [Q&A]
I deleted a helmRelease (from git, reconciled git and kustomize, helmRelease resource has gone from cluster) and expected the corresponding deployment to be purged but it wasn't.

am I missing something here - is this expected?

**Comment by stefanprodan** [comment]
You may want to check the helm-controller logs, the controller performs a helm uninstall, if there were any errors you should see them in logs/events/alerts.
